### PR TITLE
Improve activities feed view #259

### PR DESCRIPTION
**Pull request by roosdebildt**
Improves look and view of activity feed, fixes the size of images in the updates.

### CODE REVIEW

**Comment by roosdebildt** [comment]
@alicetragedy: here you go!

**Comment by alicetragedy** [comment]
thank you! :sparkling_heart:

File: app/assets/stylesheets/application.css.sass [hunk 1]
```diff
@@ -33,7 +36,8 @@ body
     right: 0
     top: 0
     height: 66px
-    background-color: rgb(222, 57, 12)
+    //background-color: rgb(222, 57, 12)
```
Comment (carpodaster) [inline app/assets/stylesheets/application.css.sass]: Can you remove this comment? If we need to revert to it, we have it in Git :smile:

**Comment by alicetragedy** [comment]
looks great so far. :+1: 

A couple of thoughts/improvements:

- the link hover is that ugly bootstrap-dark-blue, would be great to change that to dark red or something. (of course, this would probably be teams-app-wide, as I think it's like that everywhere)
- in the `status_updates` index view, I think the "new Status Update" button could be moved to a more prominent place, like top right. 
- on the `status_updates` new view, the buttons "Create Activity" and "Back" should be aligned. Ideally, "Create" Activity could even have a color (like green) to show that it's the 'positive' action.
- the show view for a status_update does something funny if the text is too long, see image below.

screen shot 2015-07-13 at 22 19 52

- the table on the index view is pretty ugly, and the date as well as title columns are pretty narrow in comparison to the text column, but I think that involves a lot more improvements and I could open a separate issue for that.

If you agree with the changes above, I'd love it if you could add them to this pull request – after that, I'm more than happy to merge. thank you! :sparkles:

**Comment by carpodaster** [comment]
@alicetragedy You raise valid points about the `students/status_updates` scope but I think the changes went into `activities`. Or should they be valid for both controllers? `¯\_(ツ)_/¯`

**Comment by alicetragedy** [comment]
@carpodaster ooops, nevermind. :speak_no_evil: I see them as both somewhat linked, but it's true, two separate controllers and also contexts. Sorry! (I think the image resizing affected both, though).
Either way, then I think we're almost good – just that removing of comment you mentioned, and the link color (which can be done now or later, as it should be app-wide, imho).

I will move the rest of those comments to a separate issue then. thank you for pointing it out! :blush:

**Comment by alicetragedy** [comment]
(and feel free to merge when those minor changes are done)

**Comment by alicetragedy** [comment]
and omg it looks SOOOOO much better, I want to merge asap :D

**Comment by roosdebildt** [comment]
@carpodaster and @alicetragedy: done! Could you close the PR? I'm still a bit reluctant to do it myself :)

And yes, I'm planning to tackle some of the other controllers separately, might combine some parts with refactoring the CSS. Please let me know what issues there are with the views! Saves a lot of research :+1:

**Comment by alicetragedy** [comment]
@roosdebildt great!!! thank you! :confetti_ball: :tada:

**Comment by carpodaster** [comment]
:fireworks:

**Comment by roosdebildt** [comment]
oh wait, for some reason the color #eee doesn't work when it's put live I see --> so there is no border around the posts and the selection box doesn't have a background. any ideas on why it works locally but not live?

**Comment by alicetragedy** [comment]
@roosdebildt do you have a screenshot? It seems to be working for me :D

**Comment by roosdebildt** [comment]
when I tried to make a screenshot I noticed it was due to my display's contrast settings... #fail

**Comment by alicetragedy** [comment]
well, good to know that there's no :bug: to fix ;)

**Comment by roosdebildt** [comment]
true! except for the one team that has three team-members and their avatars' top-margin. what's the workflow when I want to fix some minor things?

**Comment by alicetragedy** [comment]
I noticed that too – teams are not supposed to have two members.. :angry: 
Usually, when fixing something minor (such as typos or small styling errors) we can push directly to the gh-pages branch, without making a pull request – but I will let @carpodaster confirm that.

**Comment by roosdebildt** [comment]
I just made a new PR for the minor fixes for this case. I think it's nothing big but have a look if we can merge: https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/270

**Pull request merged**